warehouse/core/src/main/java/datawave/util/AccumuloCounterSource.java

```diff
@@ -62,8 +62,7 @@ public String getNextIdentifier() {
     public boolean hasNext() {
         
         if (null == iterator) {
-            try {
-                BatchScanner scanner = connector.createBatchScanner(queryTable, connector.securityOperations().getUserAuthorizations(username), 100);
+            try (BatchScanner scanner = connector.createBatchScanner(queryTable, connector.securityOperations().getUserAuthorizations(username), 100)) {
```

Comment:
Try-with-resources will close the batch scanner after we exit the try block, which is not functionally equivalent to the existing code.  If we run with this, then the iterator that is set at the end of the try block will be defunct when we call it since the underlying scanner has been closed.  I would suppress this warning as well, or handle it a different way.

Reply:
Reverted.